### PR TITLE
Add new fields & designs

### PR DESCRIPTION
**Pull request by tdincer**
I'll add the changes in the demix later.

### CODE REVIEW

File: neurophotonics/pipeline/design.py [hunk 1]
```diff
@@ -26,6 +27,38 @@ class Design(dj.Lookup):
         {"design": "D202", "design_description": "Steer 45. 45-degree-beam"},
         {"design": "D203", "design_description": "Steer 0. 30-degree beam"},
         {"design": "D204", "design_description": "Steer 45. 30-degree-beam"},
+        {
+            "design": "D105",
+            "design_description": "30 um separation. 45-degree beam. 15000 absorption length",
```
Comment (dimitri-yatsenko) [inline neurophotonics/pipeline/design.py:32]: 50 um scatter length.

File: neurophotonics/pipeline/fields.py [hunk 2]
```diff
@@ -230,6 +243,102 @@ class ESim(dj.Lookup):
...
+        dict(
+            esim=6,
+            esim_description="Narrowed to pi/4, steer 0",
+            beam_compression=0.25,
+            y_steer=0.0,
+            emitter_width=10.0,
+            emitter_height=10.00,
+            anisotropy=0.88,
+            absorption_length=15000.0,
+            scatter_length=100.0,
```
Comment (dimitri-yatsenko) [inline neurophotonics/pipeline/fields.py:255]:
```suggestion
            scatter_length=50,
```

File: neurophotonics/pipeline/fields.py [hunk 3]
```diff
@@ -230,6 +243,102 @@ class ESim(dj.Lookup):
...
+        dict(
+            esim=7,
+            esim_description="Narrow pi/6, steer 0",
+            beam_compression=0.167,
+            y_steer=0.0,
+            emitter_width=10.0,
+            emitter_height=10.00,
+            anisotropy=0.88,
+            absorption_length=15000.0,
+            scatter_length=100.0,
```
Comment (dimitri-yatsenko) [inline neurophotonics/pipeline/fields.py:271]:
```suggestion
            scatter_length=50.0,
```

File: neurophotonics/pipeline/fields.py [hunk 4]
```diff
@@ -230,6 +243,102 @@ class ESim(dj.Lookup):
...
+        dict(
+            esim=9,
+            esim_description="Narrowed to pi/6, steer 30",
+            beam_compression=0.167,
+            y_steer=np.pi / 6,
+            emitter_width=10.0,
+            emitter_height=10.00,
+            anisotropy=0.88,
+            absorption_length=15000.0,
+            scatter_length=100.0,
```
Comment (dimitri-yatsenko) [inline neurophotonics/pipeline/fields.py:303]:
```suggestion
            scatter_length=50.0,
```

File: neurophotonics/pipeline/fields.py [hunk 5]
```diff
@@ -230,6 +243,102 @@ class ESim(dj.Lookup):
...
+        dict(
+            esim=10,
+            esim_description="Narrow to pi/4, steer 45",
+            beam_compression=0.25,
+            y_steer=np.pi / 4,
+            emitter_width=10.0,
+            emitter_height=10.00,
+            anisotropy=0.88,
+            absorption_length=15000.0,
+            scatter_length=100.0,
```
Comment (dimitri-yatsenko) [inline neurophotonics/pipeline/fields.py:319]:
```suggestion
            scatter_length=50.0,
```

File: neurophotonics/pipeline/fields.py [hunk 6]
```diff
@@ -230,6 +243,102 @@ class ESim(dj.Lookup):
...
+        dict(
+            esim=11,
+            esim_description="Narrowed to pi/6, steer 45",
+            beam_compression=0.167,
+            y_steer=np.pi / 4,
+            emitter_width=10.0,
+            emitter_height=10.00,
+            anisotropy=0.88,
+            absorption_length=15000.0,
+            scatter_length=100.0,
```
Comment (dimitri-yatsenko) [inline neurophotonics/pipeline/fields.py:335]:
```suggestion
            scatter_length=50.0,
```

**Review by dimitri-yatsenko — CHANGES_REQUESTED** [review]
The main change in the simulation was that the scatter length was reduced to 50 um.

File: neurophotonics/pipeline/fields.py [hunk 7]
```diff
@@ -230,6 +243,102 @@ class ESim(dj.Lookup):
...
+        dict(
+            esim=8,
+            esim_description="Narrow to pi/4, steer 30",
+            beam_compression=0.25,
+            y_steer=np.pi / 6,
+            emitter_width=10.0,
+            emitter_height=10.00,
+            anisotropy=0.88,
+            absorption_length=15000.0,
+            scatter_length=100.0,
```
[the remark this answers is not in the document]
Comment (tdincer) [reply neurophotonics/pipeline/fields.py:287]: Right. Can't believe how I did this

**Comment by tdincer** [comment]
@dimitri-yatsenko For the new D2's I used a minimum frame number of 5. Is this reasonable?

File: neurophotonics/pipeline/demix.py [hunk 8]
```diff
@@ -48,10 +48,17 @@ def make(self, key):
...
+
         volume = (Tissue & key).fetch1("volume")
         target_rank = 150_000 * volume  # rule of thumb
         illumination = np.identity(emission.shape[0], dtype=np.uint8)
-        nframes = max(2, int(np.ceil(target_rank / detection.shape[0])))
+
+        if design in ["D205", "D206", "D207", "D208"]:
+            baseframe = 5
+        else:
+            baseframe = 2
```
Comment (dimitri-yatsenko) [inline neurophotonics/pipeline/demix.py:60]:
```suggestion
        baseframe = 5 if design in ["D205", "D206", "D207", "D208"] else 2
```

Comment (dimitri-yatsenko) [reply neurophotonics/pipeline/demix.py]: Let's try more frames, e.g. 10

File: neurophotonics/pipeline/demix.py [hunk 9]
```diff
@@ -77,21 +81,22 @@ class Demix(dj.Computed):
...
-    total_power  : float # (uW) average
-    emitter_power : float # (uW) power when on
+    selection                 : longblob  # selected cells
+    mix_norm                  : longblob  # cell's mixing vector norm
+    demix_norm                : longblob  # cell's demixing vector norm
+    bias_norm                 : longblob  # cell's bias vector norm
+    trans_bias_norm           : longblob  # don't use. Saved just in case of wrong axis choice
+    total_power               : float # (uW) average
+    emitter_power             : float # (uW) power when on
+    detector_quantum_efficiency=Null : float
```
Comment (dimitri-yatsenko) [inline neurophotonics/pipeline/demix.py:91]: it should probably not be nulllable. 
```suggestion
    detector_quantum_efficiency=1.0: float
```

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**